api/main.go

```diff
@@ -109,91 +109,102 @@ const (
 
 var verifyKey *rsa.PublicKey
 
-func main() {
-	e := echo.New()
-	e.Use(middleware.Logger())
-
-	session, err := mgo.Dial("mongodb://mongo:27017")
-	if err != nil {
-		panic(err)
-	}
-
-	err = session.DB("main").C("devices").EnsureIndex(mgo.Index{
-		Key:        []string{"uid"},
-		Unique:     true,
-		Name:       "uid",
+func EnsureIndex(session *Session, database string, collection string, key string, unique boolean) {
```

Comment:
I'm wondering if this code is actually valid for the Go compiler. Is boolean a valid data type?